app/api/v1/helpers_v1.rb

```diff
@@ -12,11 +12,19 @@ def find_object(klass, id)
     error!("#{klass} not found", 404) unless @object = klass.constantize.find(id)
   end
 
-  def find_objects(klass, ids)
+  def find_objects(klass, params)
     klass = klass.classify
 
-    if ids.any?
-      @objects = klass.constantize.where(id: ids)
+    if params[:ids].any?
+      @objects = klass.constantize.where(id: params[:ids])
+    elsif params[:query] && params[:query].length > 0 && params[:limit] == 1
```

Comment:
I'll set the limit default to 1. The "limit" param is a placeholder for now since I only coded the ability to return a single object but it will be used once I/we have enhanced this query feature to support the returning of a collection of objects.